core/tests/puppeteer-acceptance-tests/spec/logged-in-user-tests/click-all-buttons-on-navbar.spec.ts

```diff
@@ -20,9 +20,17 @@
 import {UserFactory} from '../../puppeteer-testing-utilities/user-factory';
 import {LoggedInUser} from '../../user-utilities/logged-in-users-utils';
 import testConstants from '../../puppeteer-testing-utilities/test-constants';
+import {ConsoleReporter} from '../../puppeteer-testing-utilities/console-reporter';
 
 const DEFAULT_SPEC_TIMEOUT_MSECS = testConstants.DEFAULT_SPEC_TIMEOUT_MSECS;
 
+// Exclude the error related to Google Docs Viewer since it's from an external service
+// and cannot be controlled.
+// https://stackoverflow.com/questions/50909239/how-to-use-google-docs-viewer-with-puppeteer
```

Comment:
Use the shortlink: https://stackoverflow.com/q/50909239

Reply:
Done